tools/dev/generate_mitre_attack_technique_constants.rb

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env ruby
+# -*- coding: binary -*-
+
+# Pre-requisites:
+# Run the following command to fetch the latest MITRE ATT&CK data and add it to a JSON file called mitre_attack.json:
+#   curl -s 'https://raw.githubusercontent.com/mitre/cti/master/enterprise-attack/enterprise-attack.json' | jq '[.objects[] | select(.type == "attack-pattern")]'
```

Comment:
I don't think this saves anything to disk - was that the intent of this command? 👀